<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Kit Android Reversing V2.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="TabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Descompilar</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="DragLabel" name="label">
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="text">
            <string>Arrastra la aplicacion aqui..</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_smali">
             <property name="text">
              <string>Smali</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_java">
             <property name="text">
              <string>Java</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Compilar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="DragLabel" name="label_2">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>Arrastra la aplicacion aqui</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_compile">
           <property name="text">
            <string>Compilar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_compile_sign">
           <property name="text">
            <string>Compilar y Firmar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Firmar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>16</number>
       </property>
       <item row="0" column="0">
        <widget class="DragLabel" name="label_3">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>Arrastra la aplicacion aqui</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_sign">
           <property name="text">
            <string>Firmar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>widgets/tabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DragLabel</class>
   <extends>QLabel</extends>
   <header>widgets/draglabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
